pkg/alertmanager/statefulset.go

```diff
@@ -251,7 +251,9 @@ func makeStatefulSetSpec(logger *slog.Logger, a *monitoringv1.Alertmanager, conf
 	}
 
 	if version.GTE(semver.MustParse("0.27.0")) && len(a.Spec.EnableFeatures) > 0 {
-		amArgs = append(amArgs, fmt.Sprintf("--enable-feature=%v", strings.Join(a.Spec.EnableFeatures[:], ",")))
+		efs := make([]string, len(a.Spec.EnableFeatures))
+		copy(efs, a.Spec.EnableFeatures)
+		amArgs = append(amArgs, fmt.Sprintf("--enable-feature=%s", strings.Join(efs, ",")))
```

Comment:
Whether to consider adding new fields in CRD to manage feature-specific parameters?

Reply:
Hmmm... 🤔 It seems a bit odd that both the feature name and the feature parameters are included in the `EnableFeatures`.

Reply:
Hi @dongjiang1989 ok I will add that to alertmanagerspec. Thanks